index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dogvet</title>
+  <!-- css link -->
+  <link rel="stylesheet" href="./assets/css/main.css">
+  <!-- font awesome link -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+</head>
+<body>
+  <!-- header section -->
+  <header class="header">
+    <!-- mobile menu -->
+    <div class="mobile-menu">
+      <div class="menu-bars">
+        <i class="fa-solid fa-xmark"></i>
+      </div>
+      <ul>
+        <li class="mob-list"><a href="./pages/about.html">About</a></li>
+        <li class="mob-list"><a href="#" class="mob-link">Program</a></li>
+        <li class="mob-list"><a href="#" class="mob-link">Join</a></li>
+        <li class="mob-list"><a href="#" class="mob-link">Sponsor</a></li>
+        <li class="mob-list"><a href="#" class="mob-link">News</a></li>
+      </ul>
+    </div>
+    <!-- end of mobile menu -->
+    <!-- desktop design -->
+    <div class="top-nav">
+      <ul class="top-nav-content">
+        <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="#">Swahili</a></li>
+        <li><a href="#">My page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+        <!-- main nav -->
+          <nav class="navbar">
+            <div class="top-nav-content nav-content">
+              <div class="nav-logo">
+                <img src="./assets/images/dogLogo.png" alt="logo" class="nav-logo">
+              </div>
+              <div class="nav-link-container">
+                <ul class="nav-list">
+                  <li class="list-item"><a href="./pages/about.html" class="active">About</a></li>
+                  <li class="list-item"><a href="#">Program</a></li>
+                  <li class="list-item"><a href="#">Join</a></li>
+                  <li class="list-item"><a href="#">Sponsor</a></li>
+                  <li class="list-item"><a href="#">News</a></li>
+                  <li class="list-item-btn"><a href="#" class="link">Get a dog</a></li>
+                </ul>
+              </div>
+            </div>
+          </nav>
+        <!-- end of main nav -->
+    <!-- end of desktop design -->
+    <div class="header-main-container">
+      <img src="./assets/images/header-icon.PNG" alt="" class="circle">
+      <img src="./assets/images/chess-plane.png" alt="" class="circle-2">
+      <img src="./assets/images/cross.png" alt="" class="circle-3">
+      <img src="./assets/images/cross.png" alt="" class="circle-4">
```

Comment:
- [ ] Please make sure that all your images have an `alt` attribute, as this is part of the requirements, and it's also a good practice to keep your pages accessible.